rust_src/src/javascript.rs

```diff
@@ -1550,6 +1550,29 @@ fn handle_error(e: std::io::Error, handler: LispObject) -> LispObject {
     }
 }
 
+/// Gets the current tick rate of JavaScript.
+#[lisp_fn]
+pub fn js_get_tick_rate() -> LispObject {
+    let options = EmacsMainJsRuntime::get_options();
+    unsafe { lisp::remacs_sys::make_float(options.tick_rate) }
+}
+
+/// Sets F to be the current js tick rate. Increase this number
+/// to improve file read performance, decrease to preventing
+/// hitching and save battery life.
+#[lisp_fn]
+pub fn js_set_tick_rate(f: LispObject) -> LispObject {
+    let mut options = EmacsMainJsRuntime::get_options();
+
+    unsafe {
+        lisp::remacs_sys::CHECK_NUMBER(f);
+        options.tick_rate = lisp::remacs_sys::XFLOATINT(f);
+        EmacsMainJsRuntime::set_options(options);
+    }
+
+    lisp::remacs_sys::Qnil
```

Comment:
I think we can drop this line if we return nothing.

Reply:
Corrected.